docs/resources/rds_database.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Relational Database Service (RDS)"
+---
+
+# huaweicloud_rds_database
+
+Manages RDS Mysql database resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
```

Comment:
s/instance_id/dbName

Reply:
this should be used as intance_id, fixed